package.json

```diff
@@ -584,6 +584,7 @@
     "devDependencies": {
         "@microsoft/api-extractor": "^7.33.8",
         "@microsoft/eslint-config-azuretools": "^0.2.1",
+        "@microsoft/vscode-azext-azureauth": "^1.2.1",
```

Comment:
I assume it's a mistake that this got moved to dev dependencies?

Reply:
Yeah that's weird. Good catch